.changeset-staged/good-feet-own.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/connector-kit": minor
```

Comment:
should be patch as it has no new features?

Reply:
updated